### PR TITLE
add new file, update deadlines per PR235

### PR DESCRIPTION
**Pull request by peiying2**
#5755: MOSS update, hi-IN only
#5791: New, Privacy FAQ page; update Index page
#5793: Add form field strings to newsletter page
#5850 (replaces 5777): Privacy landing page update (no deadline)
#5857: Privacy update on Firefox channel pages (no deadline)

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -522,6 +522,11 @@
         'priority'          => 1,
         'supported_locales' => $mozillaorg,
     ],
+    'privacy/faq.lang' => [
+        'priority'          => 2,
+        'deadline'          => '2018-07-30',
+        'supported_locales' => $mozillaorg,
```
Comment (flodolo) [inline app/config/sources.inc.php]: I don't think this is correct. The privacy main page (privacy/index.lang) is for a much smaller set of locales.

Comment (peiying2) [reply app/config/sources.inc.php]: There is a page called privacy/principles.lang which supports all mozorg locales, so I thought this should be at the same level.

Comment (flodolo) [reply app/config/sources.inc.php]: Have you tried opening it? It redirects me to https://www-dev.allizom.org/internet-health/privacy-security/, which makes me think it was removed at some point.

Comment (flodolo) [reply app/config/sources.inc.php]: > Have you tried opening it? It redirects me to https://www-dev.allizom.org/internet-health/privacy-security/, which makes me think it was removed at some point.

Never mind, that's not the principles page.

Comment (peiying2) [reply app/config/sources.inc.php]: You mean this one: https://www.mozilla.org/it/privacy/principles/

Comment (flodolo) [reply app/config/sources.inc.php]: It's not discoverable without the index (I've updated the other comment in the meantime).

Maybe it makes sense to open the other page to all locales, with the caveat of not being able to translate policies. Anyhow, your call ;-)

Comment (peiying2) [reply app/config/sources.inc.php]: I hate to make decisions at this time of the day!! 
How discoverable is the "principles". I have a feeling the FAQ page could be linked from other places down the road. Can we make it an opt-in page?

Comment (peiying2) [reply app/config/sources.inc.php]: I will leave this one as is, and decide on the Index page after talking to legal. It doesn't look good for sure.

Comment (peiying2) [reply app/config/sources.inc.php]: @flodolo ok to proceed otherwise?

**Comment by flodolo** [comment]
I'm going to add the files to the mozilla.org PR and merge. Given how old it is, there's a high risk of conflicts, and it's late for me too.

**Pull request merged**